ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -945,8 +945,6 @@ topk_max_sort_index
 topk_min_sort_index
 topk_1d_i32_max_all
 topk_int64
-gelu_f32
-gelu_f64
 gelu_backprop_factor_f32
```

Comment:
Are these ones longer needed?

Reply:
I have a discussion with @jdanieck and we came to the conclusion that they are not needed anymore.

Reply:
I am referring to `gelu_backprop_factor_f32`, if not maybe we can remove it? Same as with `INTERPRETER.gelu_backprop_factor_f64`

Reply:
oh ok, I think we can remoe it